src/SearchEngine.php

```diff
@@ -137,9 +137,11 @@ public function doSearch(): array {
 	private function applyResultTranslations( array $results ): array {
 		$results = $this->doFacetTranslations( $results );
 		$results = $this->doNamespaceTranslations( $results );
+		$template = $this->config->getSearchParameter( "result template" );
+		$properties = $this->config->getResultProperties();
 
 		// Allow other extensions to modify the result
-		Hooks::run( "WikiSearchApplyResultTranslations", [ &$results ] );
+		Hooks::run( "WikiSearchApplyResultTranslations", [ &$results, &$template ,&$properties ] );
```

Comment:
I wouldn't pass `$template` and `$properties` as references

Reply:
You are right!, I'll change that

```suggestion
		Hooks::run( "WikiSearchApplyResultTranslations", [ &$results, $template ,$properties ] );
```